assets/migrations/3_migrate_data_to_events.up.sql

```diff
@@ -0,0 +1,37 @@
+
+-- Purpose: Migrate data from region tables to a central, normalized events table.
+DO $$
+DECLARE
+    region_table_name TEXT;
+    table_list TEXT[] := ARRAY['mdw', 'fra', 'sin', 'nyc', 'lax', 'lon', 'prg', 'sao', 'atl', 'hnd', 'mad', 'mia', 'mos2', 'sto', 'syd', 'sea', 'tor', 'sjo', 'ash', 'hou', 'atl2'];
+    sql_command TEXT;
+BEGIN
+    FOREACH region_table_name IN ARRAY table_list
+    LOOP
+        BEGIN
+            -- make sure the region table exists before migrating data
+            IF NOT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = region_table_name) THEN
+                RAISE NOTICE 'Table % does not exist while attempting to migrate region table. Assuming this is a clean install where this table normally does not exist (aka nothing to migrate).', region_table_name;
+                CONTINUE;
+            END IF;
+            sql_command := format('
+                INSERT INTO events (event_time, orchestrator, region_id, payload)
```

Comment:
How much data do we have in this tables? I just wonder if we expect any downtime on the SQL while executing this query?

Reply:
Livepeer.Cloud SPE does not have that data or access to provide as assessment on this.  Happy to help if we had the data.   Doing some db maintenance on the existing production instance wouldn't be a bad idea either.  I am guessing there is data in there going back for years that are not being used and could be truncated.

Reply:
Yeah, I think we (probably someone from Inc) need to double-check it before executing the migration script.